lib/task-data/tasks/dell-wsman-delete-volume-getComponent.js

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+module.exports = {
+    friendlyName: "Dell Wsman GetComponent for delete volume",
+    injectableName: "Task.Dell.Wsman.Delete.Volume.getComponent",
+    implementsTask: "Task.Base.Dell.Wsman.getComponent",
+    optionsSchema: 'dell-wsman-delete-volume.json',
+    options: {
+        volumeId: "",
```

Comment:
Could you please confirm, is the empty `volumeId` valid?

Reply:
The volumeId check will be added.